requirements.txt

```diff
@@ -2,3 +2,4 @@ conda >=4
 conda-build >=1.21.7
 anaconda-client
 mock
+requests <=2.11
```

Comment:
The `<` might be trying to force it to be `2.11.0` or `2.10.*`. Maybe just making this `requests 2.11.*` would fix the issue as it then can pin in `2.11` version it likes.